lib/facter/resolvers/linux/networking.rb

```diff
@@ -62,6 +64,32 @@ def physical(ifname)
                                                          end
           end
 
+          def duplex(interface_name)
+            # not all interfaces support this, wifi for example causes an EINVAL (Invalid argument)
+            unless interface_name.start_with?('lo')
```

Comment:
I think rubocop is suggesting to return early like:

```suggestion
            return unless interface_name.start_with?('lo')
```

and then shift everything left

Reply:
I think I've cleaned up what it was upset about and made sure there is a default case for `lo`.